packages/components/src/higher-order/with-focus-return/README.md

```diff
@@ -17,20 +17,19 @@ const EnhancedComponent = withFocusReturn( () => (
 	</div>
 ) );
 
-const MyComponentWithFocusReturn = withState( {
-	text: '',
-} )( ( { text, setState } ) => {
+const MyComponentWithFocusReturn = () => {
+	const [ text, setText ] = useState( '' );
 	const unmount = () => {
 		document.activeElement.blur();
-		setState( { text: '' } );
+		setText( '' );
 	};
 
 	return (
 		<div>
 			<TextControl
 				placeholder="Type something"
 				value={ text }
-				onChange={ ( text ) => setState( { text } ) }
+				onChange={ ( value ) => setText( value ) }
```

Comment:
Nitpick:
```suggestion
				onChange={ setText }
```

Reply:
This is the pattern used across the docs and the only reason I went with it.